DQM/HLTEvF/plugins/TriggerBxMonitor.cc

```diff
@@ -127,6 +125,7 @@ void TriggerBxMonitor::fillDescriptions(edm::ConfigurationDescriptions& descript
 
 TriggerBxMonitor::TriggerBxMonitor(edm::ParameterSet const& config)
     :  // module configuration
+      m_l1tMenuToken{esConsumes()},
```

Comment:
I believe bookHistogram is called on begin Run transitions, therefore this esConsumes call must also say it happens on begin Run.